config/locales/en.yml

```diff
@@ -316,6 +316,8 @@ en:
     subtitle: |
       Search the portal for courses, events, videos, presentations, learning pathways, handbooks...<br>
       All types of resources at all levels for leveraging <span class="text-primary">computational resources in the life sciences</span>.
+    search_placeholder: >
+      Search <%= TeSS::Config.site['title_short'] %>...
```

Comment:
Can't evaluate ERB in these files, you'll need to pass in the site title as a % variable